### ISSUE TITLE
Include host-trigger-check.sh in Dockerfile

### ISSUE THREAD

**Issue by Cyber1000** [gersilex/apcupsd-docker#2]
Hi,

I think you forgot to include host-trigger-check.sh in Dockerfile, at least the Readme says, that I should copy it from the dockercontainer to the host.

Thanks for the container, have a nice day,
Cyber1000

**Comment by gersilex** [comment]
Hi @Cyber1000 ! Thank you very much for this issue. I wouldn't add the file to the Docker image, because it is not useful inside a Docker container. I also honestly don't know why I mounted the file into the container at the `docker run` part, so I removed it.

I extended the `README.md` so that it instructs to download the `host-trigger-check.sh` from the Github repository, instead of the Docker image.

If you want, feel free to check out the new Readme and tell me if it makes you happy or something is still unclear. :)

Have a nice day as well!
